CONTRIBUTING.md

```diff
@@ -1,45 +1,18 @@
 # Contributing to the C SDK
 
-Thanks for your interest in contributing to the New Relic C SDK! We look forward to engaging with you.
+Thanks for your interest in contributing to the New Relic C SDK! 
 
-## How to Contribute
+## Archival Notice ##
+As of June 30 2022 we are archiving the C-SDK project. Please explore the [option to use OpenTelemetry](https://github.com/newrelic/docs-website/blob/develop/docs/apm/agents/c-sdk/get-started/otel_cpp_example) as an alternative to send C++ telemetry data to New Relic. For more details, see our [Explorers Hub announcement](https://discuss.newrelic.com/t/q1-bulk-eol-announcement-fy23/181744).
+
+## Best Practices for Modifying Project Forks
 
 1. Read [this contributing file](CONTRIBUTING.md). (Good job!)
 1. Read our [code of conduct](CODE_OF_CONDUCT.md).
 1. Submit a [pull request](#pull-request-guidelines) or [issue](#filing-issues--bug-reports). For pull requests, please also:
```

Comment:
Is this section applicable for archived projects? Is New Relic going to review and merge any incoming PRs?